config/settings.yml

```diff
@@ -15,5 +15,14 @@
     :allow_targeted_refresh: true
     :inventory_collections:
       :saver_strategy: batch
+    :refresh_interval: 15.minutes
 :log:
   :level_ansible_tower: info
+:workers:
+  :worker_base:
+    :event_catcher:
+      :event_catcher_ansible_tower:
+        :poll: 20.seconds
+    :queue_worker_base:
+      :ems_refresh_worker:
+        :ems_refresh_worker_ansible_tower_automation: {}
```

Comment:
Another small inconsistency, the event_catcher is `worker_type_ansible_tower` where this is `worker_type_ansible_tower_automation`

The refresh_worker here has to override the settings_name for this.